docs/platforms/python/index.mdx

```diff
@@ -19,12 +19,6 @@ Don't already have an account and Sentry project established? Head over to [sent
 
 If you prefer to follow video instructions, see [How to Install the Sentry Python SDK in 60 Seconds](https://vimeo.com/899368672).
 
-<Note>
-
-Version 2.0 of the Sentry Python SDK is now available. If you're looking to upgrade from 1.x, check out the <PlatformLink to="/migration/1.x-to-2.x/">migration guide</PlatformLink>.
-
-</Note>
```

Comment:
Removed this from the Getting Started page since if you install the SDK anew as we say on this page, you'll get a new version right off the bat, so this piece of info is not really relevant for the target audience of this page.